### PR TITLE
Update user.css

### PR DESCRIPTION
**Pull request by pronoy2108**
Fix top bar, as it has been merged with main-view/root, in latest UI revamp

### CODE REVIEW

**Review by SunsetTechuila — COMMENTED** [review]
Hello and thanks for the contribution, but I personally can't approve PR in this form. It's more of a reversal of theme changes than a fix

**Review by nimsandu — COMMENTED** [review]
@pronoy2108 can you state your point of view about this pr?

File: user.css [hunk 1]
```diff
@@ -264,7 +264,7 @@ html {
 .Root__top-bar {
   position: absolute !important;
   width: 100vw !important;
-  grid-area: none;
+  grid-area: main-view;
```
Comment (SunsetTechuila) [inline user.css:267]: and that completely breaks elements centering, also making properly centering impossible at all

**Review by SunsetTechuila — COMMENTED** [review]
If you are really looking into fixing LibraryX view (I'm not sure what you mean by latest UI revamp) incompatibility - please start from [this](https://github.com/SunsetTechuila/spicetify-bloom/tree/libraryX)

**Comment by nimsandu** [comment]
@pronoy2108 would you work regarding these reviews by @SunsetTechuila ?

@SunsetTechuila thanks

File: user.css [hunk 2]
```diff
@@ -1202,55 +1202,6 @@ button.switch {
...
-
-
-.main-entityHeader-topbarTitle,
-.main-topBar-topbarTitle {
-  font-size: 1.4rem;
-  font-weight: 700;
-  padding-left: 10px;
-}
-
-.main-playButton-PlayButton {
```
[the remark this answers is not in the document]
Comment (pronoy2108) [reply user.css:1224]: By accident, I intended to remove the topbar code only as it breaks the new UI

File: user.css (hunk 1 above)
Comment (pronoy2108) [reply user.css:267]: Main-view has been brought recently, by spotify, this makes the back and forward arrow buttons work

image
Without it

image
With it

(unfortunately, I couldn't find a way to also include userwidget in the new UI)

**Review by pronoy2108 — COMMENTED** [review]
I have fixed the code recently

**Review by pronoy2108 — COMMENTED** [review]
I have fixed the code recently

**Comment by SunsetTechuila** [comment]
Absolutely mess now

**Comment by pronoy2108** [comment]
> Absolute mess now

?

**Comment by pronoy2108** [comment]
@nimsandu Please take a look

**Comment by SunsetTechuila** [comment]
image
dude

**Comment by SunsetTechuila** [comment]
why did you just copypaste **all** the lines to the file end, duplicating **all** the content?

**Pull request closed without merging**